examples/ReceiveDemo_Simple/ReceiveDemo_Simple.ino

```diff
@@ -9,8 +9,8 @@
 RCSwitch mySwitch = RCSwitch();
 
 void setup() {
-  Serial.begin(9600);
-  mySwitch.enableReceive(0);  // Receiver on interrupt 0 => that is pin #2
+	Serial.begin(115200);
+	mySwitch.enableReceive(digitalPinToInterrupt(2));  // Receiver on interrupt 0 => that is pin #2
 }
```

Comment:
Using `digitalPinToInterrupt` is of course the right thing to do here. But your commit does more than the commit message indicates: it also changes the baud rate and the indentation. That's bad. Please either amend the commit message (explaining both changes), or drop the extra changes.